applications/openshift/api-server/api_server_authorization_mode/rule.yml

```diff
@@ -2,31 +2,34 @@ documentation_complete: true
 
 prodtype: ocp3
 
-title: 'Disable use of AlwaysAllow for the API Server Authorization Mode'
+title: 'Ensure API Server authorization-mode is set to Webhook'
 
 description: |-
-    The OpenShift API server should not allow anonymous API requests.
+    By default, unauthenticated/unauthorized users have no access to OpenShift nodes
+    and the API Server <tt>authorization-mode</tt> is set to <tt>Webhook</tt>.
     To ensure that the API server requires authorization for API requests,
-    validate that <tt>AlwaysAllow</tt> is not configured for <tt>authorization-mode</tt>
+    validate that <tt>authorization-mode</tt> is is configured to <tt>Webhook</tt>
     in <tt>/etc/origin/master/master-config.yaml</tt>:
     <pre>kubernetesMasterConfig:
       apiServerArguments:
         authorization-mode:
-        - AlwaysAllow</pre>
+        - Webhook</pre>
+
+    NOTE: If <tt>authorization-mode</tt> is not configured to <tt>Webhook</tt>, the node
```

Comment:
@shawndwells please make this a warning of type `functionality` instead of a note.